integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdownMS.java

```diff
@@ -667,13 +359,13 @@ public void testShutdownOptionsOverrideClusterLevel() throws Exception {
               + " failed to create, returns null");
       // scale up to 2 replicas to check both managed servers in the cluster
       scaleCluster(2, domain);
-      Assertions.assertTrue(checkShutdownUpdatedProp(domain.getDomainUid()
+      Assertions.assertTrue(checkShutdownProp(domain.getDomainUid()
               + "-managed-server1", domain.getDomainNs(),"Graceful"),
           domain.getDomainUid()
               + "-managed-server1 :"
               + " shutdown properties don't not match the expected : "
               + "shutdownType=Graceful");
-      Assertions.assertTrue(checkShutdownUpdatedProp(domain.getDomainUid()
+      Assertions.assertTrue(checkShutdownProp(domain.getDomainUid()
```

Comment:
Add a descriptive failure message like above.

Reply:
it is there, see next line : 
 Assertions.assertTrue(checkShutdownProp(domain.getDomainUid()
              + "-managed-server2", domain.getDomainNs(),"Forced"),
          domain.getDomainUid()
              + "-managed-server2 :"
              + " shutdown properties don't not match the expected : "
              + "shutdownType=Forced");